doc/integrations/spark.ipynb

```diff
@@ -0,0 +1,872 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Spark\n",
+    "\n",
+    "This tutorial will show you how to get a Spark instance up and running locally to test JupySQL. You can run this in a Jupyter notebook. We'll use Spark Connect which is the new thin client for Spark"
```

Comment:
`test JupySQL` can be replaced with `integrate with JupySQL`

Reply:
Please add a link to Spark Connect docs like [Spark Connect](https://spark.apache.org/docs/latest/api/python/getting_started/quickstart_connect.html)

Reply:
Done